config/socket/game.js

```diff
@@ -89,6 +89,11 @@ Game.prototype.payload = function() {
   };
 };
 
+Game.prototype.broadcastNotification = function(userId) {
+  console.log('fron sockets', userId)
```

Comment:
I believe you need to remove every instance of this `console.log()` in the file you happen to have made changes to.